generatebundlefile/README.md

```diff
@@ -41,10 +41,132 @@ generatebundlefile --generate-sample
 
 This will output a sample bundle file to ./output.
 
+
+### Public Promotion to another Account
+
+```sh
+generatebundlefile --private-profile "profile-name" --input data/sample_input.yaml
+```
+
+This command will move **Only** the helm chart from the listed input files to the target **public** ECR in another account.
+
+### Private Promotion to another Account
+
+```sh
+generatebundlefile --private-profile "profile-name" --input data/sample_input.yaml
+```
+
+This command will move **Only** the images from the listed helm charts in the input files to the target **private** ECR in another account.
+
 ### Package Promotion
 
 To promote a package from a private ECR to public you need the repository name. This repository must contain a Helm chart built by the process in the eks-anywhere-build-tooling git repository.
+This will **both** for the helm chart, and the imgaes required to the public ECR of the calling user.
```

Comment:
This sentence is very confusing. Could you try re-wording it? I'd take a guess at re-wording it, but I'm really not sure what you meant.